python/taichi/linalg/sparse_cg.py

```diff
@@ -6,7 +6,7 @@
 from taichi.types import f32, f64
 
 
-class CG:
+class SparseCG:
```

Comment:
Could you add some docstring for this class and MatrixFreeCG class?

Reply:
Added docstring to both classes. Please see the updated `sparse_cg.py` and `matrixfree_cg.py` file.